lib/bullet.rb

```diff
@@ -63,6 +63,10 @@ def enable?
       !!@enable
     end
 
+    def app_root
```

Comment:
I also moved the logic to one method that can be shared. This way we're not duplicating code across bullet. I hope this was an ok change to make